vaadin-date-picker-flow-parent/vaadin-date-picker-flow-integration-tests/src/main/java/com/vaadin/flow/component/datepicker/DatePickerFormatPage.java

```diff
@@ -57,6 +67,11 @@ public DatePickerFormatPage() {
         setupSetLocaleAfterFormat();
         setupSetFormatAfterLocale();
         setupServerSideValueChange();
+        setupCustomReferenceDateAndLocaleWithTwoDigitsYear();
+        setupCustomReferenceDateAndLocaleWithFourDigitsYear();
```

Comment:
Let's move these two tests to `DatePickerLocalePage` / `DatePickerLocaleIT` as this test suite is for testing custom formats.

Maybe we should rename this suite to `DatePickerCustomFormatPage` / `DatePickerCustomFormatIT`.